Gemfile_next.lock

```diff
@@ -286,9 +285,9 @@ GEM
     mini_racer (0.2.15)
       libv8 (> 7.3)
     minitest (5.14.2)
-    monetize (1.9.4)
+    monetize (1.10.0)
```

Comment:
I checked Gemfile_next.lock in `master` and the versions don't match. It'd be safer to rebase onto `master`.

Reply:
Should be resolved by commit f8e7b76caf50a53810bbbc3700d274be97bbcdf3